scripts/zones/RoMaeve/npcs/Qu_Hau_Spring.lua

```diff
@@ -34,13 +33,13 @@ end;
 
 function onTrigger(player,npc)
     local CurrentMission = player:getCurrentMission(WINDURST);
-	local MissionStatus = player:getVar("MissionStatus");
+    local MissionStatus = player:getVar("MissionStatus");
 
-	if(CurrentMission == VAIN and MissionStatus >= 1) then
-		player:startEvent(2);
+    if(CurrentMission == VAIN and MissionStatus >= 1) then

```

Comment:
Small request both for my OCD and my bad eyesight: could you make all if `if(` consistently `if (` please? this file has both styles in it.